### ISSUE TITLE
pango needs 'pathname' loaded but doesn't require it

### ISSUE THREAD

**Issue by mvz** [ruby-gnome2/ruby-gnome2#642]
This reproduces the problem for pango 3.0.7:

```
ruby -e "require 'pango'"
```

The resulting output:

```
/home/matijs/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/pango-3.0.7/lib/pango.rb:20:in `<top (required)>': uninitialized constant Pathname (NameError)
	from /home/matijs/.rbenv/versions/2.2.3/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:128:in `require'
	from /home/matijs/.rbenv/versions/2.2.3/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:128:in `rescue in require'
	from /home/matijs/.rbenv/versions/2.2.3/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:39:in `require'
	from -e:1:in `<main>'
```

**Comment by kou** [comment]
Thanks for your report.
Could you try to fix it?

**Comment by mvz** [comment]
Sure, I'll make a pull request.

**Comment by kou** [comment]
Thanks. #584 may be related.

**Comment by mvz** [comment]
Yes, it looks like #584 is indeed related, since requiring `glib2` should take care of requiring `pathname`.

**Comment by mvz** [comment]
From #584, this turns out to be a problem in rubygems that is fixed by updating it to at least version 2.4.6.

**Comment by kou** [comment]
Thanks for confirming it!
